### PR TITLE
Remove uses of IPython.display.display

### PR DESCRIPTION
**Pull request by SylvainCorlay**
This removes all the uses of `IPython.display.display` and only uses the displaying of the last computed value as it was suggested by Fernando.

### CODE REVIEW

File: examples/Advanced Plotting/Animations.ipynb [hunk 1]
```diff
@@ -23,8 +23,7 @@
    "outputs": [],
    "source": [
     "import numpy as np\n",
-    "import bqplot as bq\n",
-    "from IPython.display import display"
+    "import bqplot as bq"
```
Comment (dmadeka) [inline examples/Advanced Plotting/Animations.ipynb]: Can we drop the as bq here too? Sorry I thought I did that

**Comment by SylvainCorlay** [comment]
Done.

**Comment by dmadeka** [comment]
@sylvaincorlay thanks! I'm only worried that people won't realize they can use display. Should we leave one example with markdown text saying that they can use display to display the plot?

**Comment by SylvainCorlay** [comment]
Although that is not really the point of bqplot... It is a jupyter thing.

**Comment by dmadeka** [comment]
Okay, so no display? Is there ever any use for display? apart from displaying one widget and then another without using a VBox? Do we ever do that in the examples?

**Comment by SylvainCorlay** [comment]
We don't do that in the examples. I removed the double displays...

**Comment by dmadeka** [comment]
Id say keep one example. If we want to display and then change without running two cells its useful

**Comment by dmadeka** [comment]
+1

**Pull request merged**